tests/python/gaia-ui-tests/gaiatest/apps/music/regions/sublist_view.py

```diff
@@ -20,9 +20,8 @@ def switch_to_active_view(self):
         self.marionette.switch_to_frame(self.marionette.find_element(*self._active_view_locator))
 
     def wait_sublist_view_draw(self):
-        active_view = self.marionette.find_element(*self._active_view_locator)
-        Wait(self.marionette).until(expected.element_displayed(active_view))
-        self.marionette.switch_to_frame(active_view)
+        Wait(self.marionette).until(expected.element_displayed(*self._active_view_locator))
+        self.marionette.switch_to_frame(self.marionette.find_element(*self._active_view_locator))
```

Comment:
wait_sublist_view_draw is used in line 44 and 51. not sure why it didn't pick it up as the diff.